enterprise/libs.amazon/external/ion-java-1.0.2-license.txt

```diff
@@ -0,0 +1,208 @@
+Name: Amazon Ion Java
+Origin: Amazon Labs
+Version: 1.0.2
+License: Apache-2.0
+Description: A Java implementation of the Amazon Ion data notation
+URL: https://github.com/amznlabs/ion-java/
```

Comment:
there is a NOTICE file in the git referenced ion-java repository, I would add it (as you already did for aws-java-sdk-core)

Reply:
NOTICE file added as `enterprise/libs.amazon/external/ion-java-1.0.2-notice.txt`